package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "accoutrement-init",
-  "version": "1.1.3",
+  "version": "1.1.4",
```

Comment:
We could avoid this breaking change, by adding `_index.scss` *in addition to* `_init.scss`.  The index file would only need to have a single line: `@import 'init';` and it should work for everyone.